src/domain/get-users.tsx

```diff
@@ -0,0 +1,19 @@
+import { useQuery } from "@apollo/client";
+import { GET_USERS } from "../data/graphql/queries/get-users-query";
+
+export const LIMIT = 10;
```

Comment:
```suggestion
export const GET_USERS_LIMIT = 10;
```
e se essa const só é usada nesse arquivo, ai pode tirar o `export`

Reply:
Ela está sendo usada em outro arquivo também 👍